pkg/controller/migration/convert/network.go

```diff
@@ -16,34 +16,50 @@ const (
 )
 
 func handleNetwork(c *components, install *Installation) error {
-	// TODO: be more elegant about creation of this spec field if it already exists
-	if install.Spec.CalicoNetwork == nil {
-		install.Spec.CalicoNetwork = &operatorv1.CalicoNetworkSpec{}
+
+	// FELIX_DEFAULTENDPOINTTOHOSTACTION
+	defaultWepAction, err := c.node.getEnv(ctx, c.client, containerCalicoNode, "FELIX_DEFAULTENDPOINTTOHOSTACTION")
```

Comment:
We should track this one, I think.

Either:
- The default is already Accept, and so we can just remove our setting of the env var.
- The default should be changed to Accept so we can remove our env var.

Then, we can simply convert this env var into datastore config if we see it. 

No work to do here now, though.